elliott/elliottlib/errata.py

```diff
@@ -764,6 +764,40 @@ def get_file_meta(advisory_id) -> List[dict]:
     return ErrataConnector()._get(f'/api/v1/erratum/{advisory_id}/filemeta')
 
 
+def create_batch(release_version, release_date):
+    """Create batch for a release
+    https://errata.devel.redhat.com/documentation/developer-guide/api-http-api.html#batches
+    :param release_name: release name, e.g. 4.14.12
+    :param release_date: release date, e.g. 2024-02-08
+    :return: return the batch id which used as parameter when creating advisory
+
+    """
+    data = {
+        "name": f"OCP {release_version}",
+        "release_name": "RHOSE ASYNC - AUTO",
+        "release_date": release_date,
+        "description": f"OCP {release_version}",
+        "is_active": True
+    }
+    response = ErrataConnector()._post("/api/v1/batches", data=data)
```

Comment:
error checking and retries would be great

Reply:
added